prime-router/src/testIntegration/kotlin/datatests/TranslationTests.kt

```diff
@@ -340,6 +381,25 @@ class TranslationTests {
                         result.passed = !readResult.actionLogs.hasErrors()
                         readResult.report
                     }
+                    else -> {
+                        result.passed = false
+                        val actionLogger = ActionLogger()
+                        actionLogger.error(
+                            InvalidReportMessage(
+                                "Format for report not handled in this test. Received $format"
+                            )
+                        )
+                        ReadResult(
+                            Report(
+                                schema,
+                                listOf(listOf("")),
+                                TestSource,
+                                null,
+                                format
+                            ),
+                            actionLogger
+                        ).report
+                    }
```

Comment:
This original case statement assumed that if it wasn't internal, HL7, or UP then treat it like CSV. I changed that to explicitly handle the CSV format, and throw an error on an different format. We should never make an assumption like this in our testing and ALWAYS code for the case we want to test.